### PR TITLE
Added spec for parsing IBM AIX utmp format

### PR DESCRIPTION
**Pull request by mariuszlitwin**
Hi, I've added spec file for parsing IBM AIX utmp format.
Based on: https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/com.ibm.aix.files/utmp.h.htm

### CODE REVIEW

File: log/aix_utmp.ksy [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    type: record
+    repeat: eos
+types:
+  record:
+    seq:
+    - id: ut_user
+      doc: User login name
+      size: 256
+      type: str
+      encoding: utf-8
```
Comment (KOLANICH) [inline log/aix_utmp.ksy]: move into `meta`

File: log/aix_utmp.ksy [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    - id: dbl_word_pad
+      type: s4
+    - id: reserved_a
+      size: 8
+    - id: reserved_v
+      size: 24
+  exit_status:
+    seq:
+    - id: e_termination
```
Comment (KOLANICH) [inline log/aix_utmp.ksy]: you don't have to keep original names in `id`. The convention is to use `-orig-id` for that.

File: log/aix_utmp.ksy (hunk 1 above)
Comment (GreyCat) [reply log/aix_utmp.ksy]: I actually seriously doubt that it's always utf-8 — I'm not even sure if AIX supports utf-8, or it is a default encoding in AIX 7.2. May be we should just let it stay without encoding?

File: log/aix_utmp.ksy [hunk 3]
```diff
@@ -0,0 +1,75 @@
+meta:
+  id: aix_utmp
+  title: utmp log file, IBM AIX version
+  license: CC0-1.0
+  endian: be
+doc: |
+  This spec can be used to parse utmp, wtmp and other similar as created by
+  IBM AIX. Spec is based on IBM Knowledge Base article:
+  https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.files/utmp.h.htm
```
Comment (GreyCat) [inline log/aix_utmp.ksy]: This URL can be moved to `doc-ref` element.

File: log/aix_utmp.ksy [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    - id: ut_id
+      doc: /etc/inittab id
+      size: 14
+      type: str
+      encoding: utf-8
+    - id: ut_line
+      doc: device name (console, lnxx)
+      size: 64
+      type: str
+      encoding: utf-8
```
Comment (GreyCat) [inline log/aix_utmp.ksy]: Most likely ASCII? I'm not sure if there's any possibility of UTF-8 characters appearing here.

File: log/aix_utmp.ksy [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    0: empty
+    1: run_lvl
+    2: boot_time
+    3: old_time
+    4: new_time
+    5: init_process
+    6: login_process
+    7: user_process
+    8: dead_process
+    9: accounting
```
Comment (GreyCat) [inline log/aix_utmp.ksy]: Missing end of file newline, please add it.

Comment (KOLANICH) [reply log/aix_utmp.ksy]: @mariuszlitwin, you may want to install [editorconfig plugin](https://editorconfig.org/#download) for your favorite editor.

**Comment by GreyCat** [comment]
Cool, thanks so much for this contribution, Mariusz! Merging in.

**Pull request merged**